src/model/Network.ts

```diff
@@ -1,26 +1,9 @@
-import Versioned from './base/Versioned';
+import { VersionedType } from './base/VersionedType';
 
-type Data = {
+export type Network = VersionedType & {
+  id?: string;
   name?: string;
   description?: string;
   privateCode?: string;
-  authorityRef: string;
+  authorityRef?: string;
```

Comment:
Må inte ett `Network` ha en referanse till en `authority`? 😊

Reply:
Jo, den må vel egentlig også ha et navn tror jeg. Men måtte sette de til optional nå for at det skal kompilere uten å endre "alle" steder. Når modellene er skrevet om så bør vi sette det som er optional til optional og required til required (Lager en sak på det) :) I mellomtiden lager vi litt større problemer for oss selv ved å følge det slavisk 😄